src/ui/components/Explorer.js

```diff
@@ -36,6 +55,9 @@ export type ExplorerState = {|
   showWeightConfig: boolean,
   view: CredView,
   recalculating: boolean,
+  anchorEl: any,
```

Comment:
Let's use HTMLElement here
```suggestion
  anchorEl: HTMLElement | null,
```